apps/aehttp/src/sc_ws_handler.erl

```diff
@@ -329,6 +329,45 @@ process_incoming(#{<<"action">> := <<"get">>,
         {error, _} ->
             {reply, error_response(R, invalid_arguments)}
     end;
+process_incoming(#{<<"action">> := <<"get">>,
+                   <<"tag">> := <<"poi">>,
+                   <<"payload">> := Filter} = R, State) ->
+    AccountsE   = maps:get(<<"accounts">>, Filter, []),
+    ContractsE  = maps:get(<<"contracts">>, Filter, []),
+    Parse =
+        fun(T, Keys) ->
+            try {ok, lists:foldr(
+                      fun(K, Acc) ->
+                              {ok, Res} = aec_base58c:safe_decode(T, K),
+                              [Res | Acc]
+                      end, [], Keys)}
+            catch
+                error:_ ->
+                    {error, invalid_pubkey}
+            end
+        end,
+    BrokenEncodingReply =
+        fun(What) ->
+            {reply, error_response(R, <<"broken_encoding: ", What/binary>>)}
```

Comment:
I don't think the error responses from failed `safe_decode` are entirely consistent throughout this API. Perhaps something to track in a PT?

Reply:
Tracked [SC WS: revisit broken encoding responses](https://www.pivotaltracker.com/story/show/159459679)